broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java

```diff
@@ -1556,7 +1553,8 @@ private RemotingCommand queryConsumeQueue(ChannelHandlerContext ctx,
         } finally {
             result.release();
         }
-
+        response.setCode(ResponseCode.SUCCESS);
```

Comment:
It would be better not to move response.setcode (ResponseCode.SUCCESS);  because when the result is null, this method will return a Response without a status code.